doc/examples/external_simulators.ipynb

```diff
@@ -0,0 +1,238 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Script based external simulators"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In this notebook, we demonstrate the usage of the script based external simulators, summary statistics, and distance functions features.\n",
+    "\n",
+    "These allow to use near-arbitrary programing languages and output for pyABC. The main concept is that all communication happens via the file system. This comes at the cost of a considerable overhead, making this feature not optimal for models with a low simulation time. For more expensive models, the overhead should be negligible."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This notebooke replicates the using_R notebook, but circumvents usage of the rpy2 package. Note the considerably longer runtime though."
```

Comment:
typo: "notebook"

Reply:
I would say instead of "Note the considerably longer runtime though." -> "For R scripts we encourage users to use r2py" (or something like this)